### ISSUE TITLE
Document Windows support

### ISSUE THREAD

**Issue by ihortom** [snowplow/schema-guru#170]
Some users are trying to run schema guru on Windows. Could we explore possibilities here?

**Comment by alexanderdean** [comment]
To be clear - it should work already on Windows; at a minimum we should improve the documentation for Windows users, potentially we could make the file executable on Windows too?

**Comment by chuwy** [comment]
Hey @ihortom! What exactly users want from Schema Guru? May be they need igluctl?

**Comment by chuwy** [comment]
Because almost everything from Schema Guru can be used in Web UI, except `ddl` command, which now is in igluctl.

**Comment by chuwy** [comment]
Yeah, that is absolutely possible with just `java -jar schema-guru-0.6.2`. I'll document it on Schema Guru and Igluctl.
